alerta-covid/config/database.yml

```diff
@@ -1,86 +1,22 @@
-# PostgreSQL. Versions 9.3 and up are supported.
-#
-# Install the pg driver:
-#   gem install pg
-# On macOS with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On macOS with MacPorts:
-#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
-# On Windows:
-#   gem install pg
-#       Choose the win32 build.
-#       Install PostgreSQL and put its /bin directory on your path.
-#
-# Configure Using Gemfile
-# gem 'pg'
-#
```

Comment:
I'd suggest not tracking the `database.yml` file to avoid changing the `development` config.
Instead of this, I'd move it to a template -> `config/database.yml.example`

That is a common configuration to allow any contributor making they preferred db-configuration 😄

Reply:
In other words: 
- Instead of making changes to this file, move it to `config/database.yml.example` and add `config/database.yml` to the `gitignore`

Reply:
Thanks for this suggestion, we will take it into account in a new issue.